applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -578,6 +578,27 @@ public function put_ban($id, array $body) {
         return $out->validate($result);
     }
 
+    /**
+     * Send a password reset email.
+     *
+     * @param array $body The POST body.
+     * @throws Exception Throws all exceptions to the dispatcher.
+     */
+    public function post_requestPassword(array $body) {
+        $this->permission();
+
+        $in = $this->schema([
+            'email:s' => 'The email/username of the user.',
+        ]);
+
+        $body = $in->validate($body);
+
+        $this->userModel->passwordRequest($body['email']);
+        $this->validateModel($this->userModel, true);
+
+        return '';
```

Comment:
No need to return anything if there's nothing to return. `null` would probably be more appropriate than an empty string. If you keep the return statement the DocBlock needs to be updated.